cloudify_cli/commands/blueprints.py

```diff
@@ -153,7 +153,7 @@ def upload(ctx,
             # When the blueprint file is already available locally, it can be
             # uploaded directly using the `upload` API call.
             logger.info('Uploading blueprint %s...', blueprint_path)
-            blueprint_obj = client.blueprints.upload(
+            client.blueprints.upload(
```

Comment:
To quote @tehasdf on the matter:
> we can but thats not very nice. a PUT on /blueprints should return a blueprint and not an execution
> we had some ideas about maybe putting a header or putting the id in the returned object somewhere
... so that's gonna be resolved later, along with similar cases in **plugins upload** and **deployments create**